### ISSUE TITLE
Microsoft comment on Linearization

### ISSUE THREAD

**Issue by alexswli** [w3c/wcag21#184]
Implementable:

a. This SC can be helpful for text content.  We believe it is met without effort from the author when you use reading view from the browser.  If so, we don't think we need to impose a new SC on content authors.

b. If the WG intends to apply this beyond text, we run into a conflict where most navigation bars and ribbons are arranged horizontally.  I think it will be a very poor user experience if we rearrange such features into a single column.  Is it the intention that content such as Office 365 ribbon would fall within the exception?  If so, the exception text does not make it clear.

**Comment by WayneEDick** [comment]
Thank you for this thoughtful observation.
I would reference you to:
http://nosetothepage.org/Fitz/2dScroll.html

The operational cost of horizontal scrolling is much greater than most accessibility experts realized. This more than overshadows the usability gains of task panes, tool bars and panels. 

For one who does not do real work in large print, a read view seem reasonable. However, most web pages don't just require reading. The jump back and forth from read mode to normal mode is more than difficult. Also many pages do not support a read mode.

Regarding panels as in MS Word. Word manages this well, because the panel can be closed to enable access to sufficient space to enter and edit text. 

Linearization has more than one meaning:
In a web page role a physical linearization of content is a physical geometric layout.

If Word was a web app, then linearization means a temporal display of single purpose windows. For low  vision the tool pane would be much more useful as a full-screen state that could be opened and closed as needed.

For full sighted users the panels in word were a real time saver. For a person with low vision they are space wasters. It is point of view. What is usable by one group is unusable by another.

**Comment by mraccess77** [comment]
@alexswli wrote a. This SC can be helpful for text content. We believe it is met without effort from the author when you use reading view from the browser. If so, we don't think we need to impose a new SC on content authors.

Reading view in my experience does not cover all functionality of the page and there are mysteries of when and how the reading view works and doesn't work.  Reading views are getting better but this just proves that page content can be transformed and linearized.

**Comment by DavidMacDonald** [comment]
@alexswli 
It appears reading mode may be very helpful for reading large volumes of text. However, interacting with a web page with low vision is a different story. Turning on and off reading mode is not practical for regular interaction.

We have exceptions for essential layout... would that address your concerns?

**Comment by alexswli** [comment]
Perhaps I do not follow the response.  But it sounds like @WayneEDick is saying that hiding the ribbon is a way of "linearizing" the content.  By hiding the ribbon, it is taking away certain easily reachable functionalities in order to give the editing surface more screen real estate.  Hiding does not linearize the ribbon.

Important to note that Word is more than a desktop app.  It is also a web app and mobile apps, like Google Docs.

The bottom line for now is: I can't visualize what a linearized Word web app looks like.  Once I know what you're asking us to do, then perhaps I can figure out if that is feasible and further extrapolate it to see how it applies in other text and non-text scenarios such as email, social media, video streaming, etc.

**Comment by steverep** [comment]
Recommend this issue be closed as the Linearization SC did not reach working group consensus and thus will not be in 2.1.  Much of the desired content requirements, however, are covered by the Zoom Content SC.

**Comment by awkawk** [comment]
@alexswli With the changes in https://www.w3.org/TR/WCAG21/#zoom-content your comment on requiring linearization of web pages is no longer applicable to the current version so closing this issue.
